kuksa_apps/kanto/README.md

```diff
@@ -0,0 +1,6 @@
+# Publish kuksa.val data via Eclipse Kanto
+// TODO
+# Prerequisites
+// TODO
+# Run the example
```

Comment:
At the moment it is not obvious to me what is "reusable code" and what is "example code", i.e. code that typically needs to be changed before it can be used in a project. I am thinking of the files referring specific VSS-signals. Would it be an idea to list the files where changes typically are needed?